packages/nextclade-web/src/io/fetchSingleDatasetAuspice.ts

```diff
@@ -0,0 +1,38 @@
+import { isEmpty } from 'lodash'
+import { FatalError } from 'next/dist/lib/fatal-error'
+import { attrStrMaybe, AuspiceTree, Dataset, DatasetFiles } from 'src/types'
+import { removeTrailingSlash } from 'src/io/url'
+import { axiosFetch } from 'src/io/axiosFetch'
+
+export async function fetchSingleDatasetAuspice(datasetJsonUrl_: string) {
+  const datasetJsonUrl = removeTrailingSlash(datasetJsonUrl_)
+
+  const auspiceJson = await axiosFetch<AuspiceTree>(datasetJsonUrl)
+  const pathogen = auspiceJson.meta?.extensions?.nextclade?.pathogen
+
+  if (isEmpty(auspiceJson.root_sequence?.nuc)) {
```

Comment:
Using the dataset at [staging/avian-flu/h5n1-cattle-outbreak/genome/nextclade-jh-no-extensions](https://nextstrain.org/staging/avian-flu/h5n1-cattle-outbreak/genome/nextclade-jh-no-extensions) ([test nextclade URL](https://nextclade-git-feat-ref-and-ann-from-tree-json-nextstrain.vercel.app/?dataset-json-url=https://nextclade-git-feat-ref-and-ann-from-tree-json-nextstrain.vercel.app/?dataset-json-url=https://nextstrain.org/staging/avian-flu/h5n1-cattle-outbreak/genome/nextclade-jh-no-extensions)) , which has a (correctly formatted) `root_sequence` in it, I get the following error:

```console
Auspice JSON does not contain required field '.root_sequence.nuc': https://nextclade-git-feat-ref-and-ann-from-tree-json-nextstrain.vercel.app/?dataset-json-url=https://nextstrain.org/staging/avian-flu/h5n1-cattle-outbreak/genome/nextclade-jh-no-extensions
```

Note that this is the _same_ JSON file as other examples which work (see other threads in this PR) but for this example I removed the "optional" `.meta.extensions` block.

Reply:
@jameshadfield  Should be fixed in [`9f3c1e0` (#1455)](https://github.com/nextstrain/nextclade/pull/1455/commits/9f3c1e0a7667c72ed9ba0e9701aedbc1e711defb)